tests/nested/manual/fde-on-classic/mk-image.sh

```diff
@@ -224,7 +224,7 @@ set timeout=3
 # the bootenv
 load_env --file /EFI/ubuntu/grubenv kernel_status snapd_extra_cmdline_args snapd_full_cmdline_args
 
-set snapd_static_cmdline_args='console=ttyS0 console=tty1 console=tty1 panic=-1'
+set snapd_static_cmdline_args='console=ttyS0 console=tty1 panic=-1'
```

Comment:
Why is this being removed now?

Reply:
It's actually not needed (to duplicate it), I wanted to clean along the way but I can remove it if it's confusing.

Reply:
Ok, I got a bit confused now because I thought this was the latest fix to make the fakeinstaller test run.

Reply:
I'm looking into the nested fakeinstaller failure right now but  right now I'm puzzled and suspect it's something in master and not in this branch that breaks it. But spread still running so hard to say right now.